src/memefactory/shared/utils.cljs

```diff
@@ -58,3 +58,9 @@
                                       :reg-entry.status/whitelisted
                                       :reg-entry.status/blacklisted)
     :else :reg-entry.status/whitelisted))
+
+(defn file-write [filename content & [mime-type]]
+  (js/saveAs (new js/Blob
+                  (clj->js [content])
+                  (clj->js {:type (or mime-type (str "application/json;charset=UTF-8"))}))
```

Comment:
here's json, but we're saving edn. should be there `application/plain`

Reply:
Done